eva-lib/src/main/java/uk/ac/ebi/variation/eva/lib/datastore/DBAdaptorConnector.java

```diff
@@ -64,7 +69,36 @@ public static VariantSourceDBAdaptor getVariantSourceDBAdaptor(String species)
         return new VariantSourceMongoDBAdaptor(getCredentials(species, properties),
                 properties.getProperty("eva.mongo.collections.files"));
     }
-    
+
+    public static MongoClient getMongoClient(Properties properties) throws UnknownHostException {
+
+        String authenticationDb = properties.getProperty("eva.mongo.auth.db", null);
+        if (authenticationDb == null || authenticationDb.isEmpty()) {
+            return new MongoClient();
```

Comment:
Even without an authentication DB, a `MongoClient` should still be initialized with the hosts and and credentials.